### ISSUE TITLE
associate .tstudio extension with app

### ISSUE THREAD

**Issue by neutrinog** [unfoldingWord-dev/ts-android#355]
http://stackoverflow.com/questions/3760276/android-intent-filter-associate-app-with-file-extension
This will allow users to import a archive by clicking on it without opening the app first.

**Comment by neutrinog** [comment]
Please display a confirmation dialog asking the user if they would like to import the project. Do the same if there is a conflict.

**Comment by neutrinog** [comment]
Did some preliminary research on this and stubbed out some code in the manifest and the home activity but it is not working.

**Comment by neutrinog** [comment]
This is working now. 
The next step will be to display a confirmation to the user. If they click yes then we will import the translation

**Comment by neutrinog** [comment]
@PhotoNomad0 This is already built and available in the Home Activity (I think). We need to display a dialog that asks the user if they would like to import or restore. If there's no conflicting target translation then we can just display a single confirmation button since restore is the only option.

**Comment by neutrinog** [comment]
note: this allows users click on a `.tstudio` file on their android device using a file browser. doing so will open the app and pass in the path to the file.

**Comment by neutrinog** [comment]
this actually only seems to work when clicking on a `.tstudio` file in ES File Explorer app. The native file browser app does not work.

**Comment by PhotoNomad0** [comment]
@neutrinog - looks like I have it working on Lenovo.  Did notice that the built in file manager chokes on file names that have more that one period.  It uses everything after the first period as the extension.  Doing some more testing now on different devices.

**Comment by PhotoNomad0** [comment]
Not working on Kitkat, checking into that.

**Comment by PhotoNomad0** [comment]
@neutrinog - here are the results of testing app association with tstudio file on the 5 devices I have:

ES File Explorer - works with all the devices I have.

Google does not provide a file manager with Android.  Each manufacturer may or may not provide their own: http://www.howtogeek.com/202644/how-to-manage-files-and-use-the-file-system-on-android/

Because of this I suspect we will see a variety of behaviors on devices.  What I saw:

- Lenovo TAB 2 - works with built-in file-manager
- NeuTab N7 Pro - KitKat 4.4.2 - built in file manager  first prompts asking the type of the file.  It gives choices of text, image, audio, or video.  I selected image and then it gave choice of apps including tstudio.  Can then launch the app.
- Asus ZenPad 8 - Android 5.0.2  - built in file manager  first prompts asking the type of the file.  It gives choices of text, image, audio, or video.  I selected image and then it gave choice of apps including tstudio.  Can then launch the app.
- LG G Pad (5.0.1) - default file manager - says file type is not supported.  No option of associating file.
